projects/igniteui-angular/src/lib/grids/grid-column-resizing.service.ts

```diff
@@ -54,11 +54,8 @@ export class IgxColumnResizingService {
      */
     get restrictResizeMin(): number {
         const columnLayoutMultiplier = this.column.grid.hasColumnLayouts ? this.column.gridColumnSpan : 1;
-        const actualMinWidth = parseFloat(this.column.minWidth);
-        const defaultMinWidth = parseFloat(this.column.defaultMinWidth) * columnLayoutMultiplier;
-
-        let minWidth = Number.isNaN(actualMinWidth) || actualMinWidth < defaultMinWidth ? defaultMinWidth : actualMinWidth;
-        minWidth = minWidth < parseFloat(this.column.width) ? minWidth : parseFloat(this.column.width);
+        const actualMinWidth = parseFloat(this.column.minWidth) * columnLayoutMultiplier;
```

Comment:
The columnLayoutMultiplier should be omitted in my opinion when used directly on user set minWidth because it is multiplied by the span of the column. For example if column has span 2 and user sets minWidth to 200px, the min width would be actually 400px.
It was used only on the default value since it is per single span and when a column spans 2 columns it should be multiplied by 2 but when users sets its value it should take exactly his value.

@dkamburov @StefanIvanov What do you think?

Reply:
I'd rather delegate the comment on this to @simeonoff

Reply:
I'd rather have @simeonoff share his opinion

Reply:
After further discussion with @simeonoff, @rkaraivanov and @skrustev we've decided to remove the `columnLayoutMultiplier` in general. The spec will be updated accordingly. Remove ` a columns has a span 2 for example, its default min width is 2 * '80px'. Similarly for span 3 it is 3 * '80px`